main.py

```diff
@@ -1,140 +1,144 @@
+char_to_trinary = {
+    " ": "000000",
+    "a": "000012",
```

Comment:
In Morse code, "a" is "dot-dash". Adding zero means it will have more interval in the front of the character.

Reply:
This dictionary is for converting between text to trinary only. The previous encoding is preserved for converting to Morse.

Reply:
Even text to trinary contains a lot of zeros in between. The previous dictionary was meant to replace dash, dot and the interval between them with 0,1,2 to the international standard